e2e_tests/integration/bolt.spec.js

```diff
@@ -85,7 +85,12 @@ describe('Bolt connections', () => {
         'contain',
         'noroles'
       )
-      cy.get('[data-testid="user-details-roles"]').should('contain', '-')
+      if (Cypress.config('serverVersion') === 4.0) {
+        cy.get('[data-testid="user-details-roles"]').should('contain', '-')
+      }
+      if (Cypress.config('serverVersion') === 4.1) {
+        cy.get('[data-testid="user-details-roles"]').should('contain', 'PUBLIC')
```

Comment:
This is a strange value for us to get back. 
Let's look into what we should show the user at some later point.